Makefile

```diff
@@ -1,4 +1,4 @@
-PLUGIN_VERSION=2.0.0
```

Comment:
Usually the version is bumped during release, so no need to do it in a feature/bugfix branch

Reply:
Tnaks @andreybavt, So should I just merge the branch to master an dthe version number will be the same ?

Reply:
could you actually roll back the version change in this branch? As discussed previously a version bump should only happen during release process and if we commit it now and merge, then during release we'll bump it again and thus skip one version

Reply:
Fixed in 243819c